packages/ui/src/helpers/statusValueCheck.ts

```diff
@@ -0,0 +1,47 @@
+type StatusProps = 'idle' | 'doing' | 'review' | 'done';
+type StatusCase = {
+  statusText: string;
+  timeStatus: string;
+  dateOrPoints: string;
+  isDoing: boolean;
+};
+
+export const statusValueCheck = (status: StatusProps, points: number, pointsMax: number, dateProps: Date) => {
+  let statusCase: StatusCase;
+
+  switch (status) {
```

Comment:
🤔 To make it more readable you can use early returns instead of the `switch` and value overwrites.